src/pages/geotag.js

```diff
@@ -150,28 +150,38 @@ export class GeotagPage extends Component {
         <Helmet title={`${geotag.name} posts on `} />
         <Header is_logged_in={is_logged_in} current_user={current_user}>
           <HeaderLogo small />
-          <Breadcrumbs className="header__breadcrumbs">
-            <Link title="All Geotags" to="/geotag">
-              <TagIcon inactive type={TAG_PLANET} />
-            </Link>
-            {!isEmpty(geotag.continent) &&
-              <Tag
-                inactive={geotag.type != 'Continent'}
-                name={geotag.continent.name}
-                type={TAG_LOCATION}
-                urlId={geotag.continent.url_name}
-              />
-            }
-            {!isEmpty(geotag.country) &&
-              <Tag
-                inactive={geotag.type != 'Country'}
-                name={geotag.country.name}
-                type={TAG_LOCATION}
-                urlId={geotag.country.url_name}
-              />
-            }
-            <Tag name={geotag.name} type={TAG_LOCATION} urlId={geotag.url_name} />
-          </Breadcrumbs>
+          <div className="header__breadcrumbs">
+            <Breadcrumbs>
```

Comment:
why did you add div here? I don't know if it's right or wrong, so I need some description